addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -58,14 +65,64 @@ def _default_default_message(self):
     chatbot_script_count = fields.Integer(string='Number of Chatbot', compute='_compute_chatbot_script_count')
     rule_ids = fields.One2many('im_livechat.channel.rule', 'channel_id', 'Rules')
 
+    _concurrent_session_greater_than_zero = models.Constraint(
+        "CHECK(concurrent_session = 'limited' OR concurrent_session_number > 0)",
```

Comment:
I'm not sure about this constraint. I think you meant 

 ```suggestion
        "CHECK(concurrent_session = 'unlimited' OR concurrent_session_number > 0)",
```
But even with this correction, shouldn't it always be greater than zero? Default is 10 anyway so if its not limited,  it should not be changed.

Reply:
You don't want the user to input a negative number here.
I think it should just check the `concurrent_session_number`, you never want something negative or 0 here. regardless of whether it's enabled or not.

Reply:
That's what I was suggesting^^